### ISSUE TITLE
Guppy v6.0.1 with megalodon Rerio model running error on GPU

### ISSUE THREAD

**Issue by liuyangzzu** [nanoporetech/megalodon#243]
I encountered error when running modification command using Guppy v6.0.1, megalodon 2.4.1 on GPU (Guppy v6.0.1 CPU can running ok), command is below:
```
megalodon ecoli_data_from_meteore.tar.untar1 --overwrite --mod-motif m CG 0 --outputs per_read_mods mods per_read_refs --mod-output-formats bedmethyl wiggle --write-mods-text --write-mod-log-probs --guppy-server-path /projects/li-lab/yang/workspace/nanome/related_project/guppy_v61/ont-guppy/bin/guppy_basecall_server --guppy-config res_dna_r941_min_modbases_5mC_v001.cfg --guppy-params '-d ./rerio/basecall_models/' --reference reference_genome/ref.fasta --processes 2 --devices 0

```

Error logs:
```
2022-01-26 22:52:01.003843 [guppy/message] ONT Guppy basecall server software version 6.0.1+652ffd1, client-server API version 10.0.0
log path:            megalodon_results/guppy_log
chunk size:          2000
chunks per runner:   512
max queued reads:    2000
num basecallers:     4
num socket threads:  2
max returned events: 50000
gpu device:          cuda:0
kernel path:
runners per device:  4
2022-01-26 22:52:01.004468 [guppy/info] crashpad_handler not supported on this platform.
2022-01-26 22:52:01.004945 [guppy/info] Listening on port ipc:///tmp/abf9-9dfc-344c-3a69.
2022-01-26 22:52:01.006011 [guppy/info] lamp_arrangements arrangement folder not found: ./rerio/basecall_models/read_splitting/lamp_arrangements
2022-01-26 22:52:01.144204 [guppy/info] CUDA device 0 (compute 7.0) initialised, memory limit 34089730048B (33766244352B free)
2022-01-26 22:52:01.153841 [guppy/info] CUDA device 0 reached memory limit while trying to allocate 0 bytes, currently at 64/34089730048
2022-01-26 22:52:01.154100 [guppy/error] CUDA error at /builds/ofan/ont_core_cpp/ont_core/barcoding/barcode_cuda.cpp:77: CUDA_ERROR_INVALID_VALUE. Error initialising basecall server using port: ipc://auto. Aborting.
2022-01-26 22:52:01.154734 [guppy/message] The basecall server has shut down successfully.
```

Also, previous Guppy 5.0.16 can also work fine for megalodon. 
Would you please help me to figure out the problems? Thank you!

**Comment by marcus1487** [comment]
There is an issue with the `--guppy-params '-d ./rerio/basecall_models/'` argument. The quotes means that unix does not expand the `./` into the full path. To resolve this issue, please provide the full path (from your filesystem root `/`) to the `--guppy-params` argument to resolve this issue.

Please note also that the current recommendation for modified base calling has moved to Remora models. See the `--remora-modified-bases` argument.

**Comment by liuyangzzu** [comment]
Dear marcus1487,
I run the same command using Guppy v5.x and ont-pyguppy-client-lib==5.x without any problems on GPU.
However, after upgrade to Guppy v6, even I change to full path as you metioned, I still failed with same error logs:

command:
```
megalodon raw_fast5s --overwrite --mod-motif m CG 0\
    --outputs per_read_mods mods per_read_refs\
    --mod-output-formats bedmethyl wiggle --write-mods-text\
    --write-mod-log-probs --guppy-server-path /projects/li-lab/yang/workspace/nanome/related_project/guppy_v61/ont-guppy/bin/guppy_basecall_server \
   --guppy-config res_dna_r941_min_modbases_5mC_v001.cfg \
   --guppy-params '-d /projects/li-lab/yang/workspace/nanome/related_project/guppy_v61/run_demo/rerio/basecall_models/' \
   --reference /projects/li-lab/Nanopore_compare/nf_input/reference_genome/ecoli/Ecoli_k12_mg1655.fasta \
   --processes 2 --devices 0
```

logs:
```
2022-01-27 12:22:14.097013 [guppy/message] ONT Guppy basecall server software version 6.0.1+652ffd1, client-server API version 10.0.0
log path:            megalodon_results/guppy_log
chunk size:          2000
chunks per runner:   512
max queued reads:    2000
num basecallers:     4
num socket threads:  2
max returned events: 50000
gpu device:          cuda:0
kernel path:
runners per device:  4
2022-01-27 12:22:14.098215 [guppy/info] crashpad_handler not supported on this platform.
2022-01-27 12:22:14.098682 [guppy/info] Listening on port ipc:///tmp/e20c-1e46-f72e-19bb.
2022-01-27 12:22:14.101413 [guppy/info] lamp_arrangements arrangement folder not found: /projects/li-lab/yang/workspace/nanome/related_project/guppy_v61/run_demo/rerio/basecall_models/read_splitting/lamp_arrangements
2022-01-27 12:22:14.253892 [guppy/info] CUDA device 0 (compute 7.0) initialised, memory limit 34089730048B (33766244352B free)
2022-01-27 12:22:14.263862 [guppy/info] CUDA device 0 reached memory limit while trying to allocate 0 bytes, currently at 64/34089730048
2022-01-27 12:22:14.264107 [guppy/error] CUDA error at /builds/ofan/ont_core_cpp/ont_core/barcoding/barcode_cuda.cpp:77: CUDA_ERROR_INVALID_VALUE. Error initialising basecall server using port: ipc://auto. Aborting.
2022-01-27 12:22:14.264536 [guppy/message] The basecall server has shut down successfully.
~
```

**Comment by Kirk3gaard** [comment]
Hi @marcus1487 

Do you know if this has been fixed with Guppy V6.0.6?

Best regards
Rasmus
